sssd_test_framework/hosts/client.py

```diff
@@ -46,9 +46,10 @@ def features(self) -> dict[str, bool]:
 
             [ -f "/usr/lib64/sssd/libsss_files.so" ] && echo "files-provider" || :
             [ -f "/usr/libexec/sssd/passkey_child" ] && echo "passkey" || :
-            [ -f "/usr/bin/sss_ssh_knownhostsproxy" ] && echo "knownhostsproxy" || :
-            man sssd.conf | grep -q "user (string)" && echo "non-privileged" || :
-            man sssd-ldap | grep -q "ldap_use_ppolicy (boolean)" && echo "ldap_use_ppolicy" || :
+            [ -f "/usr/bin/sss_ssh_knownhosts" ] && echo "knownhosts" || :
```

Comment:
Looks like old feature ("knownhostsproxy") wasn't used by any test code?

Reply:
It was used in suite I am preparing. I got advice that it makes a sense to detect the new tool instead of the old one as they might be both present along each other and we have only tests that use the new one.